src/components/companies/detail/emissions-assessment/EmissionsAssessmentDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import { Text } from "@/components/ui/text";
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog";
+import type { paths } from "@/lib/api-types";
+
+type Assessment = paths["/emissions-assessment/"]["post"]["responses"][200]["content"]["application/json"]["assessment"];
+
+interface EmissionsAssessmentDialogProps {
+  isOpen: boolean;
+  onOpenChange: (open: boolean) => void;
+  assessment: Assessment | null;
+}
+
+export function EmissionsAssessmentDialog({
+  isOpen,
+  onOpenChange,
+  assessment,
+}: EmissionsAssessmentDialogProps) {
+  return (
+    <Dialog open={isOpen} onOpenChange={onOpenChange}>
+      <DialogContent className="max-w-3xl bg-black-2 text-white max-h-[90vh] overflow-hidden flex flex-col">
+        <DialogHeader>
+          <DialogTitle className="text-2xl font-light">
+            Emissions Assessment
+          </DialogTitle>
+        </DialogHeader>
+        
+        {assessment && (
+          <div className="space-y-6 overflow-y-auto pr-4">
+            <div className="flex items-center gap-2">
+              <div className={`w-3 h-3 rounded-full ${assessment.isReasonable ? 'bg-green-500' : 'bg-red-500'}`} />
+              <Text className="text-lg">
+                {assessment.isReasonable ? 'Reasonable' : 'Unreasonable'} 
+                (Confidence: {Math.round(assessment.confidence * 100)}%)
```

Comment:
Minor since it's an internal page but there is a `formatPercent` in `@/utils/localizeUnit.ts` which would make sense to use as people may be stumbling over this code and copy it to public pages.